cypress.config.js

```diff
@@ -14,6 +14,15 @@ module.exports = defineConfig({
             email: email.toLowerCase(),
             password: '12345Qwert!'
           };
+        },
+        newArticle() {
+          const randomNumber = Math.floor(Math.random(1000) * 1000);
```

Comment:
Math.random() function returns a random decimal number between 0 and 1. To generate a random integer between 0 and 1000, you should use Math.floor(Math.random() * 1000) instead of Math.floor(Math.random(1000) * 1000).